src/routers/handlers/confirm_company/index.ts

```diff
@@ -35,6 +35,8 @@ export class ConfirmCompanyHandler extends GenericHandler<ConfirmCompanyViewData
 
         const { companyStatus: status, dateOfCreation: incorporatedOn, companyName } = await this.getCompanyProfile(req, companyNumber);
 
+        this.setCompanyName(req, companyName);
```

Comment:
Should we be setting in the GET, as technically the user hasn't confirmed this is the name they want